feat(service): add nocobase template

sarthaksavvy opened the pull request:
## Changes

- feat(service): add nocobase template

### Additional Details

- Docs PR [here](https://github.com/coollabsio/coolify-docs/pull/452)

sarthaksavvy committed:
feat: add nocobase service

ShadowArcanist commented on templates/compose/nocobase.yaml:
```diff
@@ -0,0 +1,42 @@
+
+services:
+  nocobase:
+    image: "nocobase/nocobase:latest"
```
Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

ShadowArcanist reviewed (changes requested):
Target branch should be `next` not `v4.x`

alnutile commented:
@sarthaksavvy  ^^ 🙏

peaklabs-dev committed:
Merge branch 'next' into feat/add-nocobase

peaklabs-dev committed:
feat(service): improve nocobase template

- remove nocobase broken healthcheck
- fix nocobase port
- pin versions
- update postgres version
- fix formating and naming

peaklabs-dev reviewed (approved):
Thanks for the PR I improved it quite a bit also I had to remove the nocobase `healthcheck` because it was not working, if you know the correct `healthcheck` please open a new PR.
